.yamato/mobile-build-and-test.yml

```diff
@@ -0,0 +1,90 @@
+2021.2_Build_Android_player:
+  name: 2021.2 Build Android player
+  agent:
+      type: Unity::VM
+      # Any generic image can be used, no need to have Android tools in the image for building
+      # All Android tools will be downloaded by unity-downloader-cli
+      image: package-ci/win10:stable
+      flavor: b1.small
+
+  commands:
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - mkdir C:\TMP
+    - cd C:\TMP && unity-downloader-cli -u 2021.2 -c editor -c Android -w --fast
+    - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
```

Comment:
I would recommend punning to a specific UTR version then always grabbing the latest version. Just for consistency purpose. This is what we do in other places within Unity.

Reply:
Okay, I'm testing that out now.